models/retinanet/builder.py

```diff
@@ -323,19 +326,18 @@ def get_prediction(self, conv_feat, im_info):
         pre_nms_top_n = p.proposal.pre_nms_top_n
         min_det_score = p.proposal.min_det_score
 
-        cls_logit_list, bbox_delta_list = self.get_output(conv_feat)
+        cls_logit_dict, bbox_delta_dict = self.get_output(conv_feat)
 
-        cls_logit_dict = {}
-        bbox_delta_dict = {}
+        cls_score_dict = dict()
 
-        for i, s in enumerate(stride):
-            cls_logit = X.sigmoid(data=cls_logit_list[i])
-            bbox_delta = bbox_delta_list[i]
+        for s in stride:
+            cls_score = X.sigmoid(data=cls_logit_dict["stride%s" % s])
+            bbox_delta = bbox_delta_dict.pop("stride%s" % s)
 
-            cls_logit_dict.update({'cls_logit_stride%s' % s: cls_logit})
-            bbox_delta_dict.update({'bbox_delta_stride%s' % s: bbox_delta})
+            cls_score_dict["cls_score_stride%s" % s] = cls_score
+            bbox_delta_dict["bbox_delta_stride%s" % s] = bbox_delta
 
-        args_dict = {**cls_logit_dict, **bbox_delta_dict}
+        args_dict = {**cls_score_dict, **bbox_delta_dict}
```

Comment:
Redundant temp variable definition.

Reply:
used in decode_retina below

Reply:
`Decode_retina` is the only place where the temp var `args_dict` is used. It is better to directly give `cls_score_dict` and `bbox_delta_dict` to `decode_retina`.